File: app/src/main/java/ge/nikka/stclient/edk/Base64Utils.java
package ge.nikka.edk;

import android.util.Base64;
import java.io.UnsupportedEncodingException;

public class Base64Utils {
    public static String Decrypt(String encoded) {
        byte[] valueDecoded = new byte[0];
        try {
            valueDecoded = Base64.decode(encoded.getBytes("UTF-8"), Base64.DEFAULT);
        } catch (UnsupportedEncodingException e) {

        }
        return new String(valueDecoded);
    }

    public static String Encrypt(String decoded) {
        byte[] valueEncoded = new byte[0];
        try {
            valueEncoded = Base64.encode(decoded.getBytes("UTF-8"), Base64.DEFAULT);

        } catch (UnsupportedEncodingException e){

        }
        return new String(valueEncoded);
    }
}

File: app/src/main/java/ge/nikka/stclient/Base64Utils.kt
package ge.nikka.stclient

import android.util.Base64
import java.io.UnsupportedEncodingException

object Base64Utils {
    @JvmStatic
    fun Decrypt(encoded: String): String {
        var valueDecoded: ByteArray? = ByteArray(0)
        try {
            valueDecoded = Base64.decode(encoded.toByteArray(charset("UTF-8")), Base64.DEFAULT)
        } catch (e: UnsupportedEncodingException) {
        }
        return String(valueDecoded!!)
    }

    fun Encrypt(decoded: String): String {
        var valueEncoded: ByteArray? = ByteArray(0)
        try {
            valueEncoded = Base64.encode(decoded.toByteArray(charset("UTF-8")), Base64.DEFAULT)
        } catch (e: UnsupportedEncodingException) {
        }
        return String(valueEncoded!!)
    }
}
